frontend/views/components/GroupSettings/InviteLinks.vue

```diff
@@ -48,13 +48,13 @@ page-section.c-section(:title='L("Invite links")')
           )
             i.icon-ellipsis-v
         td.c-state
-          i18n.c-state-description {{ item.state.description }}
+          i18n.c-state-description {{ item.status.description }}
           i18n.c-state-expire(
-            v-if='item.state.expireInfo'
-            :class='item.state.isExpired ? "expired" : ""'
-          ) {{ item.state.expireInfo }}
+            v-if='item.status.expiryInfo'
+            :class='{ "is-expired": item.status.isExpired }'
+          ) {{ item.status.expiryInfo }}
```

Comment:
Remember, i18n is just like the L function, it only ever takes string literals. Here, having `{{ item.status.expiryInfo }}` inside is the same as doing `L(item.status.expiryInfo)`, and neither will generate translatable strings.

So if you're using a variable that already has a translated string, you should not insert it using `i18n`, but instead you are free to include it without it. Since i18n outputs a `span` tag by default, here you can have `span.c-state-expire` instead, unless you want it to be a div (or something else)

Please make sure to double-check all other i18n usage, like on line 51, etc.

Reply:
Yeah, I missed those parts. Fixed! :)